release/7zip.xml

```diff
@@ -12,35 +12,53 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program. If not, see <http://www.gnu.org/licenses/>.
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app 7-Zip
     @url https://www.7-zip.org/
-    @cleanerversion v???
-    @cleanerdate 2019-03-11
+    @url http://p7zip.sourceforge.net/
```

Comment:
http://p7zip.sourceforge.net/ was last updated February 2009. The other url is better

Reply:
I had a German documentation about the 7-Zip Linux GUI... This docu points to http://p7zip.sourceforge.net/ and I also can't find the packages on https://www.7-zip.org/, so I pointed to both...